pages/index.tsx

```diff
@@ -1,12 +1,14 @@
 // Style
-import styles from '../styles/Home.module.scss';
+import styles from "../styles/Home.module.scss";
```

Comment:
If you want to use `"` im down mind updating the prettier config and eslint config.

Reply:
Either way works. I'll update to use single quotes since that's the current way its set up